playbooks/roles/tower/defaults/main.yml

```diff
@@ -193,3 +190,9 @@ tower_email_notification_template_recipients: '{{ send_grid_email_recipients }}'
 tower_email_notification_template_host: '{{ send_grid_host }}'
 tower_email_notification_template_port: '{{ send_grid_port }}'
 
+tower_configuration_project_scm_branch: '{{ integreatly_version | default("master") }}'
+tower_credentials_project_scm_branch: '{{ integreatly_version | default("master") }}'
+integreatly_version: "master"
```

Comment:
Shouldn't be any need for `integreatly_version` to be listed here as the project branch values will auto default to `master` should `integreatly_version` not be defined as an extra param on the job run i.e. `-e integreatly_version=release-1.6.0`

Reply:
If integreatly_version isn't defined somewhere this template will fail https://github.com/integr8ly/ansible-tower-configuration/blob/master/playbooks/roles/tower/templates/tower_bootstrap_vars.yml.j2#L6 and the install survey requires the var to be defined too https://github.com/integr8ly/ansible-tower-configuration/blob/master/playbooks/roles/integreatly/templates/workflow_install_survey.json.j2#L66